modules/installation-load-balancing-user-infra.adoc

```diff
@@ -48,6 +48,7 @@ ifeval::["{context}" == "installing-restricted-networks-bare-metal"]
 :restricted:
 endif::[]
 
+:_content-type: CONCEPT
```

Comment:
Content with tables are most often REFERENCE files, so consider using that attribute instead here.